### ISSUE TITLE
bug: StatusBar isn't working on Google pixel 6 pro

### ISSUE THREAD

**Issue by rbalet** [ionic-team/capacitor#5963]
### Capacitor Version

```
Latest Dependencies:       

  @capacitor/cli: 4.3.0    
  @capacitor/core: 4.3.0   
  @capacitor/android: 4.3.0
  @capacitor/ios: 4.3.0    

Installed Dependencies:    

  @capacitor/cli: 4.3.0    
  @capacitor/android: 4.3.0
  @capacitor/ios: 4.3.0    
  @capacitor/core: 4.3.0   

[success] Android looking great! 👌
[error] Xcode is not installed
```

### Platform(s)
*  Android V13
*  Google pixel 6 pro

### Current Behavior
*  The status stay gray
*  The Bar at the bottom is over the tabs
![image](https://user-images.githubusercontent.com/44493964/193257936-6547122d-aff8-4376-973c-0724cb55cc9e.png)

### Expected Behavior
*  Transparent status bar
*  Bar at the bottom under the tabs
![image](https://user-images.githubusercontent.com/44493964/193257263-13440139-c13c-4703-92b8-938f004ca0a5.png)

### Code Reproduction
Inside the `app.component.ts`

constructor() {
  if (Capacitor.getPlatform() !== 'web') {
      await StatusBar.setStyle({ style: Style.Dark })

      if (Capacitor.getPlatform() === 'android') {
        await StatusBar.setOverlaysWebView({ overlay: true })
        await StatusBar.setBackgroundColor({ color: '#00000000' })
      }

      await SplashScreen.hide()
    }
}

### Other Technical Details

`npm --version` output: v8.19.2

`node --version` output: v16.17.1.

```typescript
    "@angular/core": "14.2.4",
    "@capacitor/status-bar": "4.0.1",
    "@ionic/angular": "6.2.9",
    //...
```

### Additional Context
I'm actually doing the migration from ionic to capacitor.  _(from @awesome-cordova-plugins to be precise)_
On the former version, it was working as expected (on the same device).

I do not have the pixel phone & therefor cannot debug it.  
That said, I tried it in an emulator & it was working just fine.

**Comment by jcesarmobile** [comment]
This is a duplicate of https://github.com/ionic-team/capacitor-plugins/issues/1160

It's a problem with the new Android 12 Splash Screen API, it doesn't allow to make changes to the status bar while the splash is still shown. It's also affecting Cordova apps, not really a Capacitor/Cordova bug.

**Comment by rbalet** [comment]
@jcesarmobile So, if I do understand it correctly I should change the status only after the I hade the splashScreen ? 
```typescript
  await SplashScreen.hide() // <-- Here 

  if (Capacitor.getPlatform() === 'android') {
    await StatusBar.setOverlaysWebView({ overlay: true })
    await StatusBar.setBackgroundColor({ color: '#00000000' })
  }
}
```

Also, does it have an influence to this part ?  
If no, should I open a new issue ? 

![image](https://user-images.githubusercontent.com/44493964/193260353-6b84ed30-aecf-4eb3-becd-4b398c45e38e.png)
